app/components/reactions/reactions.js

```diff
@@ -20,7 +24,8 @@ export default class Reactions extends PureComponent {
         highlightedReactions: PropTypes.array.isRequired,
         postId: PropTypes.string.isRequired,
         reactions: PropTypes.object.isRequired,
-        theme: PropTypes.object.isRequired
+        theme: PropTypes.object.isRequired,
+        onAddReaction: PropTypes.func.isRequired
```

Comment:
also this one, can you move it so it is sorted?

Reply:
done